app/models/git/problem_specifications.rb

```diff
@@ -1,8 +1,10 @@
 class Git::ProblemSpecifications < Git::RepoBase
-  REPO_URL="https://github.com/exercism/problem-specifications"
+  class << self
```

Comment:
Rather than using this syntax, could we use `def.self` and `cattr_reader`. I prefer defining methods on the class rather than on the eigenclass as a general rule. If there's a reason you're doing it this way that I've not understood, let me know :)

Reply:
Using `cattr_accessor` creates a class variable and not a class instance variable. When using class variables, it gets tricky when dealing with inheritance as class variables when changed in the sub class, changes the class variable in the parent class.

Reply:
@kntsoriano That makes sense. Thanks for the explanation :)